gui/src/api/index.js

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export const api = axios.create({
-  baseURL: 'http://127.0.0.1:8090/v1',
+  baseURL: 'http://127.0.0.1:8080/v1',
```

Comment:
Is it OK to specify `127.0.0.1` ?

Reply:
You're right, it should probably be `localhost`.